neuralmonkey/decoders/autoregressive.py

```diff
@@ -106,6 +106,9 @@ def __init__(self,
         self.label_smoothing = label_smoothing
         self.tie_embeddings = tie_embeddings
 
+        self.encoder_states = None
```

Comment:
Tohle vypadá, že počítáme s tím, že máme právě jeden enkodér, ale tak to být nemusí, ne?

Reply:
Víc ENkodérů